app/helpers/search_helper.rb

```diff
@@ -85,6 +85,20 @@ def ordering
     end
   end
 
+  # Transforms controller params from get request into query args for a search
+  # @return [Hash]
+  def get_query_args
```

Comment:
I can change this to a post, however I was also hoping to begin tackling this issue.
https://github.com/clinwiki-org/clinwiki/issues/87